### ISSUE TITLE
Review for the journal 8-10-2016-mysterious-guy

### ISSUE THREAD

**Issue by abruzzihraig** [abruzzihraig/road-for-revenge#11]
Any advice would be appreciated, thanks!

**Comment by 0neSe7en** [comment]
> Since I worked here there always a guy everyday I can see in the park, in witch I have to through to get in office.

`there is(was) always` 
`in which I have to get through to the office`?

through不是动词。

第二段后半部，为什么都变成过去式了？

`every morning I walk nearby the beach I always ..`

admire不是动词。是`Living like a ...`

感觉很多地方都是过去式，不太对劲。过去式一般是过去某一个时刻发生，或者做了什么事但是现在停止了。你描述的应该是一种习惯吧。

ref: https://zh.wikipedia.org/wiki/%E8%8B%B1%E8%AA%9E%E8%AA%9E%E6%B3%95#.E5.8B.95.E8.A9.9E.E7.9A.84.E6.99.82.E5.92.8C.E4.BD.93

**Comment by abruzzihraig** [comment]
@0neSe7en 
First, Actually you are mentioning one of my confusion that if I could say 'there always a guy' rather than 'there is always a guy'. Since my language sense told me for this situation, the be verb could be omitted. So I just searched Google when I wrote the sentence, then I got the autocompletion below:

![image](https://cloud.githubusercontent.com/assets/3356523/17577303/4840ce88-5fbf-11e6-8862-f0a3d093df12.png)

So I suppose it could be 'there always a guy', but I'm not sure, how do you think about it?

**Comment by 0neSe7en** [comment]
@abruzzihraig did you see the results of these autocompletion.

![image](https://cloud.githubusercontent.com/assets/2234539/17577358/26258894-5faf-11e6-835c-30f3a4d4cdb0.png)

I searched it too..

**Comment by abruzzihraig** [comment]
@0neSe7en 
Second you mentioned, yes I still can't discriminate when I should use past tense and when I should use present tense. I've read grammar book a few times just focus on different tense. So I am always confused on between word context and story context. But yes you are right, that's my carelessness, cause there are some key words like 'always' and 'every morning' indicated I should use present tense.

**Comment by abruzzihraig** [comment]
@0neSe7en On the part if I can use 'there always a guy', yes you are right! I will remember it. Thanks!

**Comment by abruzzihraig** [comment]
@0neSe7en Can I say 'I have a little bit admired on this guy'?

**Comment by 0neSe7en** [comment]
@abruzzihraig 最前面说错了，admire是动词...

**Comment by abruzzihraig** [comment]
@0neSe7en I suppose 'lives like' might be right, because it describe a kind of action, just like 'smells like' or 'taste like'

**Comment by 0neSe7en** [comment]
这个地方我也不太确定... living、lives、live...

**Comment by InfinityBlue** [comment]
> in witch I have to get through to my office.

should be

> in which

?

**Comment by InfinityBlue** [comment]
> Sometimes he read some books on the bench, if felt cold he would rub his hands constantly. In some other time

in some other time感觉有点奇怪，是不是直接other times更好？

**Comment by idiotWu** [comment]
@InfinityBlue 我的话直接 Somethimes...

**Comment by idiotWu** [comment]
@abruzzihraig @0neSe7en what about this:

> Actually I have a little bit admired on this guy - lives like a kind of artist, even though he might have nothing to be proud of, even though nobody knows him.

"even though...even though..." seems weird here 😕

**Comment by abruzzihraig** [comment]
@InfinityBlue Yes you are right, there are two typos should be fixed.

For the sentence, actually when I wrote it I had already been feeling weird. Actually I just want to express something in Chinese '一些时间...另一些时间'. You know there is a little different sense with 'Sometimes... Sometimes' @idiotWu. Because the first one indicates if he doesn't do one thing, he must be doing another thing. So I am also confused how to express this kind of sense. Maybe 'other times' is proper?

**Comment by abruzzihraig** [comment]
@idiotWu How about 'even though he might have nothing to be proud of, and nobody knows him.'?

**Comment by tmchng** [comment]
You can't reply a comment..?

This is how I would write it

> Since I worked here there is always a guy everyday I can see in the park, in which I have to get through to my office.

_Since I started working here, I see a guy everyday in the park that I have to walk through to my office._

------------------------------
> The guy seems mysterious. He always sits on the same bench which back to road. 

What do you mean by "back to road"? As in the back side is facing the road?


------------------------------
> No matter how strong wind, no matter how dense cloud, 

_No matter how strong THE wind, how dense THE cloud_


------------------------------
> every morning when I walk nearby the bench, I always could see the guy’s back, the guy’s hat and the guy’s hands are rubbing.

_every morning when I walk nearby the bench, I can always see his back, his hat, and him rubbing his hands._


------------------------------
> He always looks like a man who is thinking, he doesn’t care about anyone through the road, just shows them his back and his indifferent style.

This is a run-on sentence. Let's break it in parts:

> He always looks like a man who is thinking, he doesn’t care about anyone through the road,

Do you mean that he doesn't care about anyone walking by, or anyone who walks past him?

> just shows them his back and his indifferent style.

_he just shows them his back with an indifferent attitude_


------------------------------
> Sometimes he read some books on the bench, 

reads

> if felt cold he would rub his hands constantly.

_if it's cold he would... _


------------------------------
> Other times, I also noticed he brought some weird musical instruments.

_Other times, I also noticed THAT he brought ..._

> Fiddled with these instruments but I am sure he didn’t play them.

_He fiddled with THOSE instruments but I'm sure he didn't play them._

> He used a microphone on a stand to record the timbre from his instruments. No one actually knew what he was doing,

What do you mean by timbre? I don't think timbre is the right word.

> but I suppose he was enjoyable.

_but I supposed he was enjoying it._

------------------------------
> Actually I have a little bit admired on this guy.

_Actually I admire this guy a little._

> Living like a kind of artist, even though he might have nothing to proud, and nobody knows him. At least he knows what he have done and what he is doing. Don’t care about other people’s mind, then enjoy the relaxing life.

_He's living like an artist. Even though he might have nothing to be proud of, and nobody knows about him, "at least he knows what he has done and what he is doing."(*1). He doesn't care about what other people thinks, and he simply enjoys a relaxing life._

(*1) I'm not sure what you're trying to say with "at least he knows what he has done and what he is doing".

------------------------------

**Comment by abruzzihraig** [comment]
@0neSe7en I saw a billboard today. It says: Live like James... So I suppose it's more like imperative?
How do you think about it? @tmchng

**Comment by idiotWu** [comment]
> Live like James...

@abruzzihraig that must be an imperative sentence since verb is placed on the starting.

**Comment by abruzzihraig** [comment]
@idiotWu And, it must be original form

**Comment by abruzzihraig** [comment]
@idiotWu @0neSe7en Oh maybe the sentence 'Lives like ...' is wrong. Because if we want to use a verb as a subject, it must be a original shape such as 'Live like ...' or infinitive 'To live like ...' or gerund 'Living like ...', any other form should be a grammar error. Am I right? @tmchng

**Comment by idiotWu** [comment]
@abruzzihraig but in your essay:

> Actually I have a little bit admired on this guy. Living like a kind of artist...

What does it mean if you use "live like a kind of artist"? Imperative sentence is used for warnings, giving an order or advise, etc.

**Comment by abruzzihraig** [comment]
@idiotWu Yes you are right, it's improper here with an imperative sentence. So the sentence 'live like an artist' should be wrong.

**Comment by abruzzihraig** [comment]
@tmchng Thanks so much Tim! Your detailed comments are significant and valuable to me. And I have to say it makes me shame, because there are so many issues are just occurred on basic grammar. But I hadn't find them, it's my careless, and It actually wasted your time on those simple issues.

I was so busy these days, and I still need time to rewrite the parts you mentioned. Hope next time I won't waste your time on those basic parts. Sorry for my careless.

**Comment by tmchng** [comment]
> but I suppose he was enjoyable.
_but I supposed he was enjoying it._

I made a typo. "I suppose" is correct


---------------------


"Living like ..." could work too. You can look up how to use dashes in writing.

_Sometimes I want to be that man -- living like an artist, and not worrying about anyone's opinion._

---------------------

@abruzzihraig haha don't need to feel ashamed. I think everyone has to go through this stage when they try to write expressively in a second language. If I were to write a blog in Japanese it'd probably be much worse.

I admire your effort. Keep it going :)
